### PR TITLE
fix: pr commment bug

### PR DESCRIPTION
**Pull request by wangrunlin**
- fix pr comment bug
- replease GH_TOKEN to [GITHUB_TOKEN](https://docs.github.com/en/actions/security-guides/automatic-token-authentication)
- remove preview-ui/yarn.lock, because cloudflare not work

### CODE REVIEW

File: .github/workflows/create-pr.yml [hunk 1]
```diff
@@ -22,7 +22,7 @@ jobs:
           deno-version: 1.38.3
       - name: generate UI
         env:
-          GH_TOKEN: ${{ secrets.GH_TOKEN }}
+          GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```
Comment (Yuyz0112) [inline .github/workflows/create-pr.yml:25]: Nice, shall we add a comment includes this link to explain why this works?

https://docs.github.com/en/actions/security-guides/automatic-token-authentication

File: preview-ui/yarn.lock
Comment (Yuyz0112) [inline preview-ui/yarn.lock:1]: Could you give the error details from Cloudflare?

**Comment by Yuyz0112** [comment]
conflicts

File: preview-ui/yarn.lock
Comment (wangrunlin) [reply preview-ui/yarn.lock:1]: I have tried `npm run build` and `yarn build`.
This is true for both commands.

```
2024-03-02T08:14:34.160628Z	Cloning repository...
2024-03-02T08:14:35.350922Z	From https://github.com/idea2app/Piano-Exam-app
2024-03-02T08:14:35.351463Z	 * branch            7164e0159b82fcccd11d88cf031d1ccd311f96b8 -> FETCH_HEAD
2024-03-02T08:14:35.351589Z	
2024-03-02T08:14:35.426877Z	HEAD is now at 7164e01 [Dewhale] prompt: {   "prompt": "生成一个服务于钢琴等级考试考生的 App 首页，包含以下内容：\r\n1. 推荐课程：与考试相关的学习课程，样式为图文卡片\r\n2. 最新活动：与考试相关的辅导类讲座，样式为图文卡片\r\n3. 个性化推荐：随机展示前两条内容的条目\r\n4. 公告通知：考试安排、新课程上架等消息，样式为跑马灯\n布局调整一下，好看点",   "images": [] }
2024-03-02T08:14:35.427342Z	
2024-03-02T08:14:35.527935Z	
2024-03-02T08:14:35.528476Z	Using v2 root directory strategy
2024-03-02T08:14:35.560197Z	Success: Finished cloning repository files
2024-03-02T08:14:36.3941Z	Detected the following tools from environment: yarn@3.6.3, nodejs@18.17.1
2024-03-02T08:14:36.394768Z	Installing project dependencies: yarn
2024-03-02T08:14:37.115478Z	➤ YN0070: Migrating from Yarn 1; automatically enabling the compatibility node-modules linker 👍
2024-03-02T08:14:37.115763Z	
2024-03-02T08:14:37.200995Z	➤ YN0000: ┌ Resolution step
2024-03-02T08:14:40.761477Z	➤ YN0032: │ fsevents@npm:2.3.3: Implicit dependencies on node-gyp are discouraged
2024-03-02T08:14:42.759632Z	➤ YN0002: │ @nivo/annotations@npm:0.84.0 [bd195] doesn't provide react-dom (p7e855), requested by @react-spring/web
2024-03-02T08:14:42.75999Z	➤ YN0002: │ @nivo/arcs@npm:0.84.0 [4d09d] doesn't provide prop-types (p92568), requested by @nivo/core
2024-03-02T08:14:42.760193Z	➤ YN0002: │ @nivo/arcs@npm:0.84.0 [4d09d] doesn't provide react-dom (pe291d), requested by @react-spring/web
2024-03-02T08:14:42.760347Z	➤ YN0002: │ @nivo/axes@npm:0.84.0 [bd195] doesn't provide react-dom (pda4db), requested by @react-spring/web
2024-03-02T08:14:42.760516Z	➤ YN0002: │ @nivo/bar@npm:0.84.0 [661e2] doesn't provide prop-types (pab477), requested by @nivo/core
2024-03-02T08:14:42.760674Z	➤ YN0002: │ @nivo/bar@npm:0.84.0 [661e2] doesn't provide react-dom (p5769d), requested by @react-spring/web
2024-03-02T08:14:42.760819Z	➤ YN0002: │ @nivo/core@npm:0.84.0 [3236d] doesn't provide react-dom (pe3adb), requested by @react-spring/web
2024-03-02T08:14:42.760986Z	➤ YN0002: │ @nivo/core@npm:0.84.0 [8b69b] doesn't provide react-dom (p9227f), requested by @react-spring/web
2024-03-02T08:14:42.76112Z	➤ YN0002: │ @nivo/core@npm:0.84.0 [aa595] doesn't provide react-dom (pd320c), requested by @react-spring/web
2024-03-02T08:14:42.761257Z	➤ YN0002: │ @nivo/core@npm:0.84.0 [bd195] doesn't provide react-dom (pd7768), requested by @react-spring/web
2024-03-02T08:14:42.761411Z	➤ YN0002: │ @nivo/heatmap@npm:0.84.0 [661e2] doesn't provide prop-types (p4a85e), requested by @nivo/core
2024-03-02T08:14:42.761589Z	➤ YN0002: │ @nivo/heatmap@npm:0.84.0 [661e2] doesn't provide react-dom (pf29ad), requested by @react-spring/web
2024-03-02T08:14:42.761772Z	➤ YN0002: │ @nivo/line@npm:0.84.0 [661e2] doesn't provide react-dom (pe84da), requested by @react-spring/web
2024-03-02T08:14:42.761971Z	➤ YN0002: │ @nivo/pie@npm:0.84.0 [661e2] doesn't provide prop-types (pc94eb), requested by @nivo/core
2024-03-02T08:14:42.762128Z	➤ YN0002: │ @nivo/scatterplot@npm:0.84.0 [661e2] doesn't provide prop-types (pf3ba2), requested by @nivo/core
2024-03-02T08:14:42.762274Z	➤ YN0002: │ @nivo/scatterplot@npm:0.84.0 [661e2] doesn't provide react-dom (pab59c), requested by @react-spring/web
2024-03-02T08:14:42.762567Z	➤ YN0002: │ @nivo/tooltip@npm:0.84.0 doesn't provide prop-types (p3a776), requested by @nivo/core
2024-03-02T08:14:42.762702Z	➤ YN0002: │ @nivo/tooltip@npm:0.84.0 doesn't provide react (p26170), requested by @react-spring/web
2024-03-02T08:14:42.762939Z	➤ YN0002: │ @nivo/tooltip@npm:0.84.0 doesn't provide react (p4765c), requested by @nivo/core
2024-03-02T08:14:42.763049Z	➤ YN0002: │ @nivo/tooltip@npm:0.84.0 doesn't provide react-dom (pf77a2), requested by @react-spring/web
2024-03-02T08:14:42.763395Z	➤ YN0002: │ @nivo/voronoi@npm:0.84.0 [aa595] doesn't provide prop-types (p67a71), requested by @nivo/core
2024-03-02T08:14:42.763593Z	➤ YN0060: │ preview-ui@workspace:. provides date-fns (pf8fb6) with version 3.0.1, which doesn't satisfy what react-day-picker requests
2024-03-02T08:14:42.763799Z	➤ YN0000: │ Some peer dependencies are incorrectly met; run yarn explain peer-requirements <hash> for details, where <hash> is the six-letter p-prefixed code
2024-03-02T08:14:42.766493Z	➤ YN0000: └ Completed in 5s 566ms
2024-03-02T08:14:42.812416Z	➤ YN0000: ┌ Post-resolution validation
2024-03-02T08:14:42.812747Z	➤ YN0028: │ The lockfile would have been modified by this install, which is explicitly forbidden.
2024-03-02T08:14:42.812929Z	➤ YN0000: └ Completed
2024-03-02T08:14:42.813088Z	➤ YN0000: Failed with errors in 5s 614ms
2024-03-02T08:14:42.863932Z	Error: Exit with error code: 1
2024-03-02T08:14:42.864228Z	    at ChildProcess.<anonymous> (/snapshot/dist/run-build.js)
2024-03-02T08:14:42.8644Z	    at Object.onceWrapper (node:events:652:26)
2024-03-02T08:14:42.864549Z	    at ChildProcess.emit (node:events:537:28)
2024-03-02T08:14:42.864703Z	    at ChildProcess._handle.onexit (node:internal/child_process:291:12)
2024-03-02T08:14:42.872085Z	Failed: build command exited with code: 1
2024-03-02T08:14:45.543539Z	Failed: error occurred while running build command
```

Comment (TechQuery) [reply preview-ui/yarn.lock:1]: @Yuyz0112 I fixed CloudFlare pages deployment with migrating from Yarn to PNPM 9.

**Pull request merged**